src/HomeKit/HMEnums.cs

```diff
@@ -1136,6 +1144,19 @@ public enum HMSignificantEvent {
 		Sunset,
 	}
 
+	[iOS (13,0), TV (13,0), Watch (6,0), NoMac]
+	[Native]
```

Comment:
I don't know if I need ```[Native]```or not. Maybe I need to dig further in the documentation.